src/CISDSCResourceGeneration/functions/private/Import-RegistryPol.ps1

```diff
@@ -28,21 +27,17 @@ function Import-RegistryPol {
     }
 
     process {
-        [System.Collections.ArrayList]$ScaffoldingBlocks = @()
-
         Get-ChildItem -Path $GPOPath -Filter 'registry.pol' -Recurse | ForEach-Object -Process {
             $PolicyData = Parse-PolFile -Path $_.FullName
 
             Foreach($Policy in $PolicyData){
                 switch($_.Directory.BaseName){
-                    'Machine' {$ScaffoldingBlocks += ConvertFrom-RegistryPolGPORaw -KeyName $Policy.KeyName -ValueName $Policy.ValueName -ValueData $Policy.ValueData -ValueType $Policy.ValueType}
+                    'Machine' {ConvertFrom-RegistryPolGPORaw -KeyName $Policy.KeyName -ValueName $Policy.ValueName -ValueData $Policy.ValueData -ValueType $Policy.ValueType}
```

Comment:
I'm tripling down on suggesting reconsidering implicit return here.

Really hope I haven't just misread the code. :)